packages/cordova/package.json

```diff
@@ -78,9 +78,9 @@
     "scripts": {
         "build": "webpack",
         "start": "npm run build && cordova run",
-        "prepare": "npm run build && cordova prepare",
-        "build:android": "npm run prepare && cordova build android",
-        "build:android:signed": "npm run prepare && cordova build android --release -- --packageType=apk --keystore=./android-upload-key.keystore --storePassword=$PL_ANDROID_STORE_PASSWORD --alias=$PL_ANDROID_KEYSTORE_ALIAS --password=$PL_ANDROID_STORE_PASSWORD",
-        "build:ios": "npm run prepare && cordova build ios"
+        "prepare-build": "npm run build && cordova prepare",
```

Comment:
I forgot `prepare` was a reserved lifecycle event/script, so it was always running even if it wasn't necessary, which caused issues on windows. An automatic run for the cordova CI confirmed this still works fine.